modules/virt-nmstate-example-policy-configurations.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-nmstate-example-policy-configurations"]
+= Example policy configurations for different interfaces
+
+Before reading the different example `NodeNetworkConfigurationPolicy` (NNCP) manifest configurations, consider the following factors when applying a policy so that your cluster runs at its best performance conditions:
```

Comment:
```suggestion
Before you read the different example `NodeNetworkConfigurationPolicy` (NNCP) manifest configurations, consider the following factors when you apply a policy so that your cluster runs at its best performance conditions:
```

Reply:
:heavy_check_mark: